start-page/release/index.html

```diff
@@ -10,26 +10,37 @@
 {% block main %}
       <header>
         <h1>{{ self.title() }}</h1>
-        <p>{{ _('Thunderbird is a free and open source, privacy respecting, application built for helping you communicate with others freely and effectively. Thunderbird is built by a passionate community of developers, designers, writers, and enthusiastic users. Please take the time to read below how you can contribute to the project and help sustain and improve Thunderbird.') }}</p>
       </header>
       <div>
         <section>
-          <h2><i class="far fa-life-ring fa-lg" style="color:#0060df"></i> {{ _('<a href="%(support)s">Get Support</a>')|format(support='https://support.mozilla.org/products/thunderbird') }}</h2>
-          <p>{{ _('Need help with something in Thunderbird. We have a <a href="%(support)s">support page</a> where you ask questions and read knowledge base articles to learn how to use Thunderbird.')|format(support='https://support.mozilla.org/products/thunderbird') }}</p>
+          <h2>{{ _('Why we need donations') }}</h2>
+              <p>
+              {{ _('<strong>Thunderbird improvements are no longer paid for by Mozilla.</strong> Fortunately there is an active community keeping it running and developing it further. But to survive long term, the project needs funding.') }}
```

Comment:
Maybe not bold this. And perhaps phrase it as "Thunderbird is no longer funded by Mozilla"

Reply:
I will adjust this today.